helper/clickhouse/clickhouse.go

```diff
@@ -82,13 +102,14 @@ type loggedReader struct {
 	logger   *zap.Logger
 	start    time.Time
 	finished bool
+	queryID  string
 }
 
 func (r *loggedReader) Read(p []byte) (int, error) {
 	n, err := r.reader.Read(p)
 	if err != nil && !r.finished {
 		r.finished = true
-		r.logger.Info("query", zap.Duration("time", time.Since(r.start)))
+		r.logger.Info("query", zap.String("query_id", r.queryID), zap.Duration("time", time.Since(r.start)))
```

Comment:
Hey. Thank you for your contribution!
```suggestion
		r.logger.Error("query", zap.String("query_id", r.queryID), zap.Duration("time", time.Since(r.start)))
```

I'm pretty sure, it should be `r.logger.Error` here